packages/slate/src/changes/with-schema.js

```diff
@@ -67,42 +66,32 @@ function normalizeNodeAndChildren(change, node, schema) {
     normalizeNode(change, node, schema)
     return
   }
+  const normalizedKeys = []
```

Comment:
Can you add a line break here to match the rest of the code?

Reply:
Fixed. Thank you